lib/upgradable-multisig/multisig/validation.ak

```diff
@@ -170,24 +198,22 @@ pub fn validate_remove(
   let Transaction { inputs, extra_signatories, mint, .. } = tx
 
   expect Some(Input {
-    output_reference: in_ref,
+    output_reference: input_output_ref,
     output: Output {
       address: Address { payment_credential: Script(own_policy), .. },
-      value: in_val,
       ..
     },
   }) = inputs |> list.at(input_index)
 
-  let reference_correct = own_ref == in_ref
+  let reference_correct = own_ref == input_output_ref
   let enough_signers = signed_within_threshold(datum, extra_signatories)
 
-  let token_correct = quantity_of(in_val, own_policy, multisig_token_name) == 1
+  let multisig_token_name = utils.multisig_token_name(input_output_ref)
```

Comment:
No, this is incorrect. We cannot check the name of the token here, we should just do something like
```
expect [Pair(tkn, 1)] = dict.to_pairs(tokens(mint, own_policy))
```
as above.